dockerfile-image-update/src/main/java/com/salesforce/dockerfileimageupdate/utils/GithubAppCheck.java

```diff
@@ -76,11 +76,13 @@ public GithubAppCheck(final Namespace ns){
      */
     protected boolean isGithubAppEnabledOnRepository(String fullRepoName) {
         Integer maxRetryCount = 5;
-        Boolean isGithubAppInstalled = isGithubAppEnabledOnRepositoryWithGitApi(fullRepoName);
+        Boolean isGithubAppInstalled = false;
         if (!isGithubAppInstalled) {
             for (Integer i = 0; i < maxRetryCount; i++) {
                 isGithubAppInstalled = isGithubAppEnabledOnRepositoryWithGitApi(fullRepoName);
-                if (isGithubAppInstalled) break;
+                if (isGithubAppInstalled) {
+                    return isGithubAppInstalled;
+                }
```

Comment:
Lets make isGithubAppEnabledOnRepositoryWithRenovateApi the first call made and retry this only once.  Then fallback to isGithubAppEnabledOnRepositoryWithGitApi and retry only once.  There should be a delay between the original call and the retry.

Reply:
@cjneasbi This has been resolved